cmd/api/main.go

```diff
@@ -24,38 +22,21 @@ func init() {
 }
 
 func main() {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
 	cfg, err := models.GetConfig()
 	if err != nil {
 		log.Fatalf("loading config: %s", err)
 	}
 
-	repo, err := mongo.NewRepository(ctx, *cfg)
+	conn, err := grpc.Dial(fmt.Sprintf("%s:%s", cfg.GrpcHost, cfg.GrpcPort), grpc.WithInsecure())
```

Comment:
you might want to use `grpc.DialContext()` as you can add `context`, allowing you to add a timeout for the connection